app/client/templates/single-app/single-app.scss

```diff
@@ -60,24 +58,35 @@ $app-container-padding-rl: 4.6rem;
     }
 
     .header-row {
-      text-align: justify;
+      text-align: justify; 
```

Comment:
In this first line, the only change is that you've added a ` ` character at the end of the line, which means that it shows up in this diff view but isn't a change that you intended.

Not a show-stopper; just remarking.